index.bs

```diff
@@ -13,6 +13,7 @@ Logo: https://resources.whatwg.org/logo-console.svg
 !Participate: <a href="https://github.com/whatwg/console">GitHub whatwg/console</a> (<a href="https://github.com/whatwg/console/issues/new">new issue</a>, <a href="https://github.com/whatwg/console/issues">open issues</a>)
 !Participate: <a href="https://wiki.whatwg.org/wiki/IRC">IRC: #whatwg on Freenode</a>
 !Commits: <a href="https://github.com/whatwg/console/commits">https://github.com/whatwg/console/commits</a>
+!Version History: <a href="https://twitter.com/consolelog">@consolelog</a>
```

Comment:
I didn't know if there was a difference between Version History and Commits (and streams uses the other one :) )